apps/condo/next.config.js

```diff
@@ -104,4 +109,25 @@ module.exports = withTM(withLess(withCSS({
 
         return config
     },
+
 })))
+
+if (sentryConfig['client']) {
+    module.exports = withSentryConfig(
+        module.exports,
```

Comment:
a bit strange syntax here to use module as variable :)

Maybe something like this?
```
let config = ...
if (...) {
    config = wrapper(config)
}

module.exports = config
```